File: llm_generated_code/ChatGPT/CPP/Level1/Q18.cpp
#include <iostream>
#include <vector>
using namespace std;

int maxConsecutiveOnes(const vector<int>& nums) {
    int maxCount = 0, currentCount = 0;
    for (int num : nums) {
        if (num == 1) {
            currentCount++;
            maxCount = max(maxCount, currentCount);
        } else {
            currentCount = 0;
        }
    }
    return maxCount;
}

int main() {
    vector<int> nums = {1, 1, 0, 1, 1, 1};
    cout << maxConsecutiveOnes(nums) << endl;
    return 0;
}

File: llm_generated_code/ChatGPT/Python/Level1/Q18.py
def max_consecutive_ones(arr):
    max_count = 0
    current_count = 0
    for num in arr:
        if num == 1:
            current_count += 1
            max_count = max(max_count, current_count)
        else:
            current_count = 0
    return max_count
